### ISSUE TITLE
Noise "flash" effect in rules

### ISSUE THREAD

**Issue by gelid** [AdiAddons/AdiButtonAuras#348]
#### Which version of AdiButtonAuras are you using (enter the exact version number here)?

9.0.0
#### How did you install the addon (twitch client/manually)?

twitch client
#### If using the twitch client, what type of release did you install (alpha/beta/release)?

release
#### If you installed manually, where did you download the addon from?

--
#### Which type of installation did you choose (nolib/normal)?
just press update in client. dont understand what it mean

#### If your report is about a missing or wrong spell provide the following information:
  - Your class: rogue
  - Your spec: outlaw
  - Name of the spell (and spell id if known):
  - Source of the spell (core ability/talent/pvp):
  - Do you use a macro to cast the spell (post the macro if so):

#### If you have an error report, copy it below:

i make rule for flash finisher when i have 5 cp.
it work but flash icon so terrible. how can i fix it?

![flash noise](https://user-images.githubusercontent.com/14069982/95986346-8229e400-0e2e-11eb-90db-714c832450fc.jpg)

and some spell chane its ids
315341 instead 199804
315508 instead 193316
and slice and dice new id is 315496

**Comment by gelid** [comment]
i am try all  flash, hint, stacks, good, bad, darken,lighten, dispel
flash is noise
hint and stacks not working
good,bad work 

im prefer flash animation. 
i can't find where textures assist to flash hilglight 
may be something change in textures name 
i am try to find it in AdiButtonAuras/libs/LibButtonGlow-1.0/LibButtonGlow-1.0.lua now

**Comment by arcadepro** [comment]
If you disable all addons except Adibuttonauras does this still happen? Post a screenshot of this.

**Comment by gelid** [comment]
yes i did it.  flash is simular noise 
now wow is offline cant post screen

**Comment by gelid** [comment]
i fix it..but it is crutch 

AdiButtonAuras/libs/LibButtonGlow-1.0/LibButtonGlow-1.0.lua
line 95-96
i think
`frame.ants:SetAlpha(0)`
dont work
and i do this at 95
`frame.ants:SetSize(frameWidth * 0.1, frameHeight * 0.1)`

looking good
![2020-10-14_21-27-20](https://user-images.githubusercontent.com/14069982/96030002-19f5f500-0e64-11eb-9b20-67d5785669d3.png)

**Comment by arcadepro** [comment]
You can re-create this bug by fighting a mob with a kickable spell. I also have seen it now.

**Comment by Rainrider** [comment]
An easier way to re-create this is to create the following custom rule:
```lua
return Configure {
    'testFlash',
    'Test flash marching ants',
    2983, -- Sprint
    'player',
    'UNIT_AURA',
    function(units, model)
        model.flash = true
    end
}
```
Which will always show the flash on Sprint.

However the issue is again that I updated too early for the packager to pick the updated LibButtonGlow version. This should be resolved in version 9.0.1.

Please re-open this if you are still able to reproduce it.
